adal/src/main/java/com/microsoft/aad/adal/TokenCacheItem.java

```diff
@@ -226,6 +232,23 @@ public final void setFamilyClientId(final String familyClientId) {
         this.mFamilyClientId = familyClientId;
     }
 
+    public final void setExtendedExpiresOn(final Date extendedExpiresOn) {
+        this.mExtendedExpiresOn = extendedExpiresOn;
+    }
+
+    public final Date getExtendedExpiresOn() {
+        return this.mExtendedExpiresOn;
```

Comment:
this. is not required unless the paramter and class variable name is same. Since we prepend m to all class variable, this. is never required.
about the rest of the file, we will clean it up with time.

---
In reply to: [71436590](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/676#discussion_r71436590) [](ancestors = 71436590)